assets/images/Logo.svg

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Before putting SVG into project, you can use npm package called `svgo` in order to compress the size of file
https://www.npmjs.com/package/svgo